new/LIT7141Story.xml

```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7141Story" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Story of the son of the king of the island of Nāmulāʼas</title>
+                <title xml:lang="gez" xml:id="t2">ዜናሁ፡ ለወልዱ፡ ንጉሥ፡ ዘደሴተ፡ ናሚላአስ</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Zenāhu la-waldu nǝguś za-daseta Nāmulāʼas</title>
```

Comment:
Transcription and fidal must be identical. Either ናሙላአስ፡ = Nāmulāʾas or ናሚላአስ፡ Nāmilāʾas
End fidal with ፡
Use proper transcription sign ʾ for Aleph.